init.fsx

```diff
@@ -184,4 +196,44 @@ let generate templatePath generatedFilePath =
 
 generate (localFile "build.template") (localFile "build.fsx")
 generate (localFile "docs/tools/generate.template") (localFile "docs/tools/generate.fsx")
+
+//Handle source control
+let isGitRepo () =
+  try
+    let info = Git.CommandHelper.findGitDir __SOURCE_DIRECTORY__
+    info.Exists
+  with
+    | _ -> false
+
+let setRemote (name,url) workingDir =
+  try
+    let cmd = sprintf "remote add %s %s" name url
+    match Git.CommandHelper.runGitCommand __SOURCE_DIRECTORY__ cmd with
+    | true ,_,_ -> tracefn "Successfully add remote '%s' = '%s'" name url
+    | false,_,x -> traceError <| sprintf "Couldn't add remote: %s" x
+  with
+    | x -> traceException x
+
+let isRemote (name,url) value =
+  let remote = getRegEx <| sprintf @"^%s\s+%s\s+\(push\)$" name url
+  remote.IsMatch value
+
+let isScaffoldRemote = isRemote ("origin","http://github.com/fsprojects/ProjectScaffold.git")
```

Comment:
Sure. The idea being: if a user has cloned this from `fsprojects`, then the init script should nuke the git history. This prevents users from accidentally "acquiring" the scaffold history. It also prevents (me mostly) accidentally pushing commits to scaffold that are for some other project. The other thing about this approach (though of lesser importance) is that if some has forked scaffold before cloning it, then we don't delete their history. I'm not certain if it's useful... but it feels like it could be.